x/authz specs (#8499)
cosmos/cosmos-sdk: x/README.md

Here are some production-grade modules that can be used in Cosmos SDK applications, along with their respective documentation:

- [Auth](auth/spec/README.md) - Authentication of accounts and transactions for Cosmos SDK application.

> - [Authz](authz/spec/README.md) - Allows accounts to grant authorizations to perform actions on behalf of that account to other accounts.

Reviewer: ```suggestion
- [Authz](authz/spec/README.md) - Authorization for accounts to perform actions on behalf of other accounts.
```
